Feature/dm 4867/lwm2m notifications2 operation subs behavior announcement (#3704)
SoftwareAG/c8y-docs: content/change-logs/device-management/lwm2m-agent-1021-10-30-notifications2-operation-subs-behavior-change.md

---

> date: 

Author: ```suggestion
date: 2025-09-18
```

> title: LWM2M Service utilization of Notifications 2.0 tenant-level operation subscription

Reviewer: ```suggestion
title: LWM2M service utilization of Notifications 2.0 tenant-level operation subscription
```

product_area: Device management & connectivity
change_type:
  - value: change-inv-3bw8e

>     label: Announcement

Reviewer: @Oezge-Akin This is not an announcement but an improvement/feature, right? Announcements are actually only used for upcoming changes.

Author: I was thinking as announcement for the behavior change but we can also make it feature type.

Author: Updated now to the feature type

>   - value: change-inv-3bw8e
>     label: Announcement

Author: ```suggestion
  - value: change-QHu1GdukP
    label: Feature
```

component:
  - value: component-1KLUzmqfe
    label: LWM2M
build_artifact:
  - value: tc-ggH2M4hf3
    label: lwm2m-agent
ticket: DM-4867
version: 1021.10.30
---

> LWM2M Service is now utilizing {{< product-c8y-iot >}}'s Notifications 2.0 tenant-level operation subscriptions to retrieve the device operations in real-time. This enhances the service's overall performance.

Reviewer: ```suggestion
The LWM2M service is now utilizing {{< product-c8y-iot >}}'s Notifications 2.0 tenant-level operation subscriptions to retrieve device operations in real time. This enhances the service's overall performance.
```

> Please note that when this mode is active, device push connection will no longer appear as active in the Device info widget.

Reviewer: ```suggestion
Note that when this mode is active, device push connections will no longer appear as active in the **Info** tab of the device details.
```

> Please note that when this mode is active, device push notifications will no longer appear as active in the Device info widget.

Author: ```suggestion
Please note that when this mode is active, device push connection will no longer appear as active in the Device info widget.
```
